locations/spiders/chaussea_fr.py

```
@@ -0,0 +1,19 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, apply_category
+from locations.structured_data_spider import StructuredDataSpider
+from locations.user_agents import BROWSER_DEFAULT
+
+
+class ChausseaFRSpider(SitemapSpider, StructuredDataSpider):
+    name = "chaussea_fr"
+    item_attributes = {"brand": "Chausséa", "brand_wikidata": "Q62082044"}
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)